scripts/tasks/fluentui-publish.js

```diff
@@ -1,91 +1,30 @@
-  const fuiPackageJsonFiles = glob.sync('packages/fluentui/*/package.json', { cwd: root });
-  for (let packageJsonFile of fuiPackageJsonFiles) {
-    const fullPath = path.join(root, packageJsonFile);
-    const packageJson = fs.readJSONSync(fullPath);
-    if (!denyList.includes(packageJson.name)) {
-      packageJson.private = flag;
-    }
-    fs.writeJSONSync(fullPath, packageJson, { spaces: 2 });
-  }
-}
-
-function setLernaPackages(root, packages) {
-  const lernaJsonFile = path.join(root, 'lerna.json');
-  const lernaJson = fs.readJSONSync(lernaJsonFile);
-  lernaJson.packages = packages;
-  fs.writeJSONSync(lernaJsonFile, lernaJson, { spaces: 2 });
-
-  const packageJsonFile = path.join(root, 'package.json');
-  const packageJson = fs.readJSONSync(packageJsonFile);
-
-  if (packageJson.workspaces) {
-    packageJson.workspaces.packages = packages;
-    fs.writeJSONSync(packageJsonFile, packageJson, { spaces: 2 });
-  }
-}
-
-function gitTag(root) {
-  const lernaJsonFile = path.join(root, 'lerna.json');
-  const lernaJson = fs.readJSONSync(lernaJsonFile);
-  const version = lernaJson.version;
-  const tag = `fluentuizero_v${version}`;
-  spawnSync('git', ['tag', '-a', '-f', tag, '-m', tag], { cwd: root });
-
-  logger.info(
-    `Release commit successfully tagged. To push the changes, run 'git push --no-verify --follow-tags --verbose origin HEAD:master'`
-  );
-}
-
-module.exports.fluentuiPrepublish = function() {
-  const root = findGitRoot();
-  setPrivateFlagOnFluentuiPackages(root, false);
-  setLernaPackages(root, ['packages/fluentui/*']);
-};
+const findGitRoot = require('../monorepo/findGitRoot');
 
-module.exports.fluentuiPostpublish = function() {
-  const root = findGitRoot();
-  setPrivateFlagOnFluentuiPackages(root, true);
-  setLernaPackages(root, ['apps/*', 'packages/*', 'scripts', 'packages/fluentui/*']);
-  spawnSync('git', ['add', 'packages/fluentui', 'lerna.json'], { cwd: root });
-  spawnSync('git', ['commit', '-m', 'bumping @fluentui packages'], { cwd: root });
+module.exports.fluentuiPrepublish = function() {};
 
-  gitTag(root);
-};
+module.exports.fluentuiPostpublish = function() {};
 
 module.exports.fluentuiLernaPublish = function(bumpType) {
   return function() {
-    const root = findGitRoot();
-
+    const fluentRoot = path.resolve(findGitRoot(), 'packages', 'fluentui');
     const lernaPublishArgs = [
       'lerna',
       'publish',
       "--tag-version-prefix='fluentuizero_v'",
       '--no-git-reset',
-      '--no-push',
-      '--no-git-tag-version',
       '--registry',
       argv().registry,
       bumpType
     ];
 
-    logger.info(`Running this command: npx ${lernaPublishArgs.join(' ')}`);
+    logger.info(`Running this command: yarn ${lernaPublishArgs.join(' ')}`);
 
-    const result = spawnSync('npx', lernaPublishArgs, {
-      cwd: root,
+    const result = spawnSync('yarn', lernaPublishArgs, {
```

Comment:
I don't know why `npx` should be used as we use `yarn` 🤔

Reply:
Does the yarn version still work if you don't have lerna globally installed? I'm guessing that was the reason for using npx.

Reply:
https://classic.yarnpkg.com/en/docs/cli/#locally-installed-clis

Oops, however I see what you mean. Are there any reasons why `lerna` is not a dependency under `fabricui/scripts`?

Reply:
Oh cool, didn't know that. I'm guessing lerna isn't a dep there because it's only used at the root, but @kenotron would know better.

Reply:
I just had been using npx locally that knew it worked. But if yarn works, I'm fine with that change.

Reply:
@ecraig12345 Oh, I see. `lerna` is under root `package.json`. It's also fine, `yarn` will work in this case 👍